calico-cloud/get-started/prepare-cluster.mdx

```diff
@@ -0,0 +1,209 @@
+---
+description: Prepare your cluster to install Calico CLoud.
+---
+
+# Prepare your cluster for Calico Cloud
+
+Get your cluster ready to connect to Calico Cloud.
+
+## Prerequisites
+
+* Your cluster meets the [system requirements](system-requirements.mdx) for Calico Cloud.
+
+## Allow outbound traffic from pods to Calico Cloud endpoints
+
+Pods running in your Kubernetes cluster must allow outbound traffic to the following endpoints:
+
+- `https://installer.calicocloud.io:443/*`
+- `https://www.calicocloud.io:443/api/*`
+- `https://client-auth.calicocloud.io:443/*`
+- TCP to `<management-cluster>.calicocloud.io:9000`
+
+For each node, Docker must be able to pull images from the following endpoints:
+
+* `quay.io`
+* `cdn01.quay.io`
+* `cdn02.quay.io`
+* `us-docker.pkg.dev`
+
+## Confirm that your cluster is not managed by a Kubernetes reconciler
+
+:::tip
+This limitation may not apply to certain AKS clusters with Addon-manager.
+To see whether your you can install Calico Cloud on your AKS cluster with Addon-manager, see [Verify AddonManager is compatible on AKS cluster](#verify-addonmanager-AKS)
+:::
+
+Calico Cloud cannot be installed on clusters that are managed by any kind of Kubernetes reconciler (for example, Addon-manager).
+To verify, look for an annotation called `addonmanager.kubernetes.io/mode` on either of the following resources. (The resources may not exist).
+
+- `tigera-operator` deployment in the `tigera-operator` namespace (Installation on AKS may still work if the mode is Reconcile, see [AKS](aks.mdx#verify-addonmanager-is-compatible) for an additional check.)
+- `calico-node` daemonset in the `kube-system` namespace
+
+If the following command finds addonmanager on either of the resources, then Addon-manager is being used. Find a different cluster to use.
+
+```bash
+kubectl get <type> -n <namespace> <resource-name> -o yaml | grep ' addonmanager.kubernetes.io/mode:'
+```
+
+
+## Optional: Verify settings for your managed Kubernetes service
+
+Although {{prodname}} checks that your cluster meets [System requirements](system-requirements.mdx), verifying now can save you time troubleshooting.
+
+### Verify settings for EKS
```

Comment:
@tmjd This is new. Surely the same checks would apply to EKS as to the other platforms? For IAM permissions, probably some config verification? Anything else to add here?

We've had customers think that EKS is not supported because this was missing.

Reply:
Something like that is reasonable  but I don't think the link you've found is around permissions in an EKS cluster but permissions in the AWS console accessing EKS features.